dbt_sugar/core/connectors/postgres_connector.py

```diff
@@ -0,0 +1,55 @@
+"""
+Module Postgres connector.
+
+Module dependent of the base connector.
+"""
+from typing import List
+
+import sqlalchemy
+from base import BaseConnector
+
+
+class PostgresConnector(BaseConnector):
+    """
+    Connection class for Postgres.
+
+    Child class of base connector.
+    """
+
+    def __init__(self, user: str, password: str, database: str, host: str = "localhost") -> None:
+        """
+        Init method to instanciatee the credentials.
+
+        Args:
+            user (str): user name.
+            password (str): password.
+            database (str): database name.
+            host (str): host name.
+        """
+        self.connection_string = f"postgresql://{user}:{password}@{host}/{database}"
+
+    def generate_connection(self) -> sqlalchemy.engine:
```

Comment:
sqlaclhemy is a bit funny I feel. I thought `create_connection` returned a `sqlalchemy.engine.Engine` object but apparently this seems to work too?